src/VisualStudio/Core/Def/Implementation/LanguageClient/AlwaysActivateInProcLanguageClient.cs

```diff
@@ -39,7 +37,7 @@ internal class AlwaysActivateInProcLanguageClient : AbstractInProcLanguageClient
             ILspWorkspaceRegistrationService lspWorkspaceRegistrationService)
             : base(languageServerProtocol, workspace, diagnosticService: null, listenerProvider, lspWorkspaceRegistrationService, diagnosticsClientName: null)
         {
-            _globalOptionService = globalOptionService;
+            _ = globalOptionService;
```

Comment:
I don't know whether the removal of the parameter from the ctor can affect code relying on IVT. So I kept it with a discard to silence the warning. If it's known that this ctor can safely be modified, let me know.

Reply:
Yes, you can safely remove the parameter of a MEF importing ctor.

Reply:
📝 Our use of `[Obsolete]` on these constructors ensures they can be altered.